torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -2432,6 +2455,75 @@ def test_callback_in_rpc(self):
         )
         self.assertEqual(ret, torch.ones(2, 2) + 1 + 2)
 
+    @dist_init
+    def test_future(self):
+        f = rpc.Future()
+        f.set_result(torch.ones(2, 2))
```

Comment:
We should probably add some torchscript tests for this once the Futures are merged.